docs/en/anticorona_drones.md

```diff
@@ -0,0 +1,72 @@
+## Drones to fight Corona
+
+### Team
+- Daria Miklashevskaya
+- Yuriy Sukhorukov
+
+Innopolis University, B17-DS-II, B17-RO-I
+
+
+### Introduction
+* The world faces the worst pandemic of XXI century, which affects lifes and well-being of millions of citisens. To slow down the spread of the disease and give the healthcare system to react, people are obliged to wear masks, however, people sometimes ignore those rules, which puts lifes of others under the threat. 
+There are laws in place to enforce mask wearing, but there is just not enough cops to monitor the situation in every mall, bus station etc and fine the law-breakers. We want to contribute to the solution of this problem, but instead of punishing people, we want to provide them with mask with our super-friendly and cute drone.
+
+### Custom Airframe
+* Since we do not have the Clover drone, we will use an airframe built and designed by us.
+
+* The main idea is to use truss structure, because it works well against twisting and warping 
+![](https://i.imgur.com/NpX6YnA.png)
```

Comment:
Please, don't use external images, commit them to the repo instead.

Reply:
Done!